templates/kubeconeurope2020/index.html

```diff
@@ -27,13 +27,16 @@ <h5 class="p-muted-heading u-align--center">
       <div>
```

Comment:
Can you add https://vanillaframework.io/docs/utilities/embedded-media to improve the responsive behaviour.